app/assets/javascripts/alchemy/alchemy.internal_links_select.js.coffee

```diff
@@ -0,0 +1,35 @@
+class window.Alchemy.InternalLinksSelect
```

Comment:
In Addition to the comment below we should rename this class into `InternalAnchors`